### ISSUE TITLE
bug display last 1000 pages...

### ISSUE THREAD

**Issue by onlinejo** [francodacosta/grav-plugin-page-stats#36]
I have a bug when I click on : "Display last 1000 pages".
Usage: user/data/page-data.sqlite (18.4 MB)

This is where it happens: "admin/page-stats/recently-viewed-pages"
And I've got a blank page with information but not complete without my site's theme, so the information is unreadable...
-----

Server Error
Sorry, something went terribly wrong!

E_ERROR - Maximum execution time of 30 seconds exceeded
For further details please review your logs/ folder, or enable displaying of errors in your system configuration.

June 2nd at 4:57am	CRITICAL	Maximum execution time of 30 seconds exceeded

**Comment by francodacosta** [comment]
Feels like we are showing too much data, I will look into adding pagination

**Comment by onlinejo** [comment]
yes, I think it would be good, I tried to make changes to not lose too much data but it bug completely, so I came to you, if you do that it would be cool!
